app/main.py

```diff
@@ -1,3 +1,55 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
```

Comment:
The type hint `int | float` is not valid in Python versions below 3.10. If you are using an earlier version, consider using `Union[int, float]` from the `typing` module.